packages/files/_test.pony

```diff
@@ -189,10 +194,37 @@ class iso _TestSymlink is UnitTest
       target_path.symlink(link_path),
       "could not create symbolic link to: " + target_path.path)
 
+
+class iso _TestFilePathFrom is UnitTest
+  fun name(): String => "files/FilePath.from-success"
+  fun apply(h: TestHelper) =>
+    try
+      let path = "tmp.filepath"
+      let ambient = h.env.root as AmbientAuth
```

Comment:
both this and the other test I think would be better as

```pony
match h.env.root
| let auth: Ambient Auth =>
    ... logic...
else
    h.fail("env.root isn't AmbientAuth.")
end
```

as they are currently constructed, the `as` is within the `try` block which makes the cause of an error unpredictable.
i think the only thing that can error within the `try` should be the `FilePath.from` call.